foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -0,0 +1,55 @@
+%define _binaries_in_noarch_packages_terminate_build 0
+
+Name: foreman-bootloaders-redhat
+Version: 201705231433
+Release: 1%{?dist}
+Summary: Binaries of Grub1, Grub2 and Shim bootloaders
+
+Group: Applications/System
+License: GPLv2+ and GPLv3+ and BSD
+URL: https://github.com/theforeman/foreman-bootloaders
+Source0: http://downloads.theforeman.org/foreman-bootloaders/%{name}-%{version}.tar.bz2
+BuildArch: noarch
+
+Requires: /var/lib/tftpboot
```

Comment:
I take it back, both Fedora and RHEL are really `/var/lib/tftpboot`. That's also what we use in our installer (see tftp/manifests/params.pp). Also in the proxy it's the same default value:

```
config/settings.d/tftp.yml.example
5:#:tftproot: /var/lib/tftpboot
```

I think you have it wrong, unless you use Debian/Ubuntu where this path is different I believe @mmoll ?